Automattic/wp-calypso
# Sibyl: use separate support blogs for wpcom and jetpack sites

Fixes https://github.com/Automattic/wp-calypso/issues/22223

`happychat_support_blog` becomes `jetpack_support_blog` and `wpcom_support_blog`, and we use the correct one based on the `jetpack` property of the site.

How to test:

Go to `/help/contact` and select a jetpack blog, then enter the word "domains" in the contact form. No questions should appear. 

Go to `/help/contact` and select a wpcom blog, then enter the word "domains" in the contact form. Domain related questions should appear.
